src/languages/nl-NL.js

```diff
@@ -138,48 +138,48 @@ export default {
     "Two Factor Authentication": "Two Factor Authenticatie",
     Active: "Actief",
     Inactive: "Inactief",
-    "Also apply to existing monitors": "Also apply to existing monitors",
+    "Also apply to existing monitors": "Voeg ook toe aan bestaande monitors",
     Token: "Token",
     "Show URI": "Toon URI",
     "Clear all statistics": "Wis alle statistieken",
-    retryCheckEverySecond: "Retry every {0} seconds.",
-    importHandleDescription: "Choose 'Skip existing' if you want to skip every monitor or notification with the same name. 'Overwrite' will delete every existing monitor and notification.",
-    confirmImportMsg: "Are you sure to import the backup? Please make sure you've selected the right import option.",
+    retryCheckEverySecond: "Probeer elke {0} seconds.",
+    importHandleDescription: "Kies 'Sla bestaande over' als je elke monitor of melding met dezelfde naam wilt overslaan. Kies 'Overschrijf' als je elke monitor of notificatie wilt verwijderen.",
+    confirmImportMsg: "Weet je zeker dat je dit bestand wilt importeren?",
     "Heartbeat Retry Interval": "Heartbeat Retry Interval",
-    "Import Backup": "Import Backup",
-    "Export Backup": "Export Backup",
-    "Skip existing": "Skip existing",
-    Overwrite: "Overwrite",
-    Options: "Options",
-    "Keep both": "Keep both",
-    Tags: "Tags",
-    "Add New below or Select...": "Add New below or Select...",
-    "Tag with this name already exist.": "Tag with this name already exist.",
-    "Tag with this value already exist.": "Tag with this value already exist.",
-    color: "color",
-    "value (optional)": "value (optional)",
-    Gray: "Gray",
-    Red: "Red",
-    Orange: "Orange",
-    Green: "Green",
-    Blue: "Blue",
+    "Import Backup": "Importeer Backup",
+    "Export Backup": "Exporteer Backup",
+    "Skip existing": "Sla bestaande over",
+    Overwrite: "Overschrijf",
+    Options: "Opties",
+    "Keep both": "Bewaar beide",
+    Tags: "Labels",
+    "Add New below or Select...": "Voeg nieuwe toe of selecteer...",
+    "Tag with this name already exist.": "Label met deze naam bestaat al",
+    "Tag with this value already exist.": "Label met deze waarde bestaat al",
+    color: "Kleur",
+    "value (optional)": "waarde (optioneel)",
+    Gray: "Grijs",
+    Red: "Rood",
+    Orange: "Oranje",
+    Green: "Groen",
+    Blue: "Blauw",
     Indigo: "Indigo",
-    Purple: "Purple",
-    Pink: "Pink",
-    "Search...": "Search...",
-    "Avg. Ping": "Avg. Ping",
-    "Avg. Response": "Avg. Response",
+    Purple: "Paars",
+    Pink: "Roze",
+    "Search...": "Zoeken...",
+    "Avg. Ping": "Gemiddelde Ping",
+    "Avg. Response": "Gemiddelde Response",
     "Entry Page": "Entry Page",
-    statusPageNothing: "Nothing here, please add a group or a monitor.",
+    statusPageNothing: "Nothing here, please add a group or a monitor. Niets hier, voeg een groep of monitor toe.",
```

Comment:
Left the English in on this one.

Reply:
I will fix this, Thanks for looking into it.